src/server/utils.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
+
+const fs = require('fs')
+const path = require('path')
+
+function mkdirRec (p, cb) {
+  fs.mkdir(p, (err) => {
+    if (!err || err.code === 'EEXIST') {
```

Comment:
There might be more errors to handle, depending on the chosen path: EACCES, EPERM and maybe ELOOP.